src/Statistics.jl

```diff
@@ -86,18 +86,22 @@ Compute the mean using the specified `method`.
         method::GradientMethod;
         x0=x[1],
         stop_iter=100,
-        kwargs...
+        retraction::AbstractRetractionMethod = ExponentialRetraction(),
+        inverse_retraction::AbstractInverseRetractionMethod = LogarithmicInverseRetraction(),
+        kwargs...,
     )
 
 Compute the mean using the gradient descent scheme [`GradientMethod`](@ref).
 
 Optionally, provide `x0`, the starting point (by default set to the first data
-point). Set `x0` to `nothing` to use the intitial value of `y` as the starting
-point. `stop_iter` denotes the maximal number of iterations to perform and the
+point). `stop_iter` denotes the maximal number of iterations to perform and the
 `kwargs...` are passed to [`isapprox`](@ref) to stop, when the minimal change
 between two iterates is small. For more stopping criteria check the
 [`Manopt.jl`](https://manoptjl.org) package and use a solver therefrom.
 
+Optionally, pass `retraction` and `inverse_retraction` method types to specify
```

Comment:
Similar to the nice idea in the Symmetric it would maybe be nice to point to `AbstractRetractionMethod` and  `AbstractInverseRetractionMethod`s here?

Reply:
That would be nice. They're currently not in our docs at all. I'm assuming because they're not exported by ManifoldsBase.

Reply:
That should probably be resolved in another PR.